### ISSUE TITLE
Create/fork TODO example app?

### ISSUE THREAD

**Issue by ryankask** [reflux/refluxjs#30]
This project really interests me and I'm integrating it into my app.

It would be great, however, to demonstrate the "reflux way" using the the TODO app as an example. (perhaps just forking https://github.com/facebook/flux/blob/master/examples/flux-todomvc)

For example, I'm particularly interested in the best way to set initial state for components. In the Facebook TODO app example, they defined a function that gets called when the store emits its change event:

``` js
function getTodoState() {
  return {
    allTodos: TodoStore.getAll(),
    areAllComplete: TodoStore.areAllComplete()
  };
}
```

In the Reflux examples I've seen, state is passed passed directly to the callback rather than the component calling store methods. Of course the same thing could be done with Reflux but I'm not sure if that would be missing the point.

In short, it would be useful to see how the author of the project uses the library in a more detailed way with React.

**Comment by pierrepinard** [comment]
@ryankask: I also wondered about authentication in a react / react-router / reflux app, and I found this quite rich example: https://github.com/WRidder/react-spa , which may interest people with such concerns

**Comment by srph** [comment]
^Thanks! :+1:

**Comment by ivan-kleshnin** [comment]
My daily-updated conceptual demo-app https://github.com/Paqmind/react-starter
Not "production-ready" yet, but may be useful I think.
